source/frontend/src/features/mapSideBar/hooks/usePropertyDetails.ts

```diff
@@ -13,9 +13,12 @@ import {
 
 export function usePropertyDetails(property?: Api_Property): IPropertyDetailsForm | undefined {
   const isMounted = useIsMounted();
-  const electoralService = useAdminBoundaryMapLayer();
-  const legalAdminService = useLegalAdminBoundariesMapLayer();
-  const firstNationsService = useIndianReserveBandMapLayer();
+  const electoralService_ = useAdminBoundaryMapLayer();
```

Comment:
nit: could make this a single line right? this electoralservice line isn't used elsewhere.

Reply:
yes it could. I will revisit this again to make sure the memoing is actually working as I expected